packages/components/addon/components/hds/form/field/index.js

```diff
@@ -10,6 +10,8 @@ export const LAYOUT_TYPES = ['vertical', 'flag'];
 export default class HdsFormFieldIndexComponent extends Component {
   @tracked ariaDescribedBy = getAriaDescribedBy(this);
   @tracked descriptors = [];
+  @tracked isRequired = this.args.isRequired;
```

Comment:
[issue] I don't see where this is also adding the `required` attribute to the input element if true.

Reply:
If we are providing a way for the author to say "isRequired" - and that provides a required badge, do you think it would be logical to also add the `required` attribute to the associated `input` element?

Reply:
Since we've decided to leave responsibility to the consumers to handle most part of the HTML attributes for the controls, especially validation, I think we should follow the same approach to `required` here: if a field is required and the consumers handle if and how to validate that the field is "filled" by the user, if and how to show an error, then adding a `required` attribute would go against this approach, potentially may be unexpected too (we may have client-side validation interfering with the consumers-validation, at UI level).

Reply:
Sure, I'll get ambassadors' view on this.